state-of-the-docs-2015.rst

```diff
@@ -0,0 +1,197 @@
+.. post:: Sep 7, 2015
+   :tags: report
+   :author: Anthony
+
+State of the Docs
+=================
+
+August has historical significance for Read the Docs, so it seemed fitting to
+wrap up August by taking a moment to step back and reflect on what we've done in
+the last year.
+
+*What makes August so significant for us?*
+
+For starters, this August saw the **5th birthday of Read the Docs**!  We're
+still humbled by the project's success, and the important role it continues to
+play for old and new users alike.
+
+This growth has been occasionally difficult to manage, especially when there
```

Comment:
If we want this to be more of a celebratory post, we can. I wrote this strictly as a review though, so yeah, some parts are more negative for the sake of transparency.

This could be reworded to convey that the solution to handling growth is being on the project full time, not  part time.